### ISSUE TITLE
Remove getPref helpers when they are obsolete.

### ISSUE THREAD

**Issue by wolfbeast** [MoonchildProductions/UXP#989]
With #988 implementing support for a default value for `get*Pref()`, the getPref helpers scattered in our tree will become obsolete and should be removed, and replaced with direct calls to `get*Pref()`

Depends on #988, so filing this as on hold until that is complete.

**Comment by wolfbeast** [comment]
http://xref.palemoon.org/uxp-trunk/search?string=getpref%28%22
